Source/Swift/EmptyDataSetSource.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  EmptyDataSetSource.swift
+//  EmptyDataSet
+//
+//  Created by Ignacio Romero Zurbuchen on 2020-01-01.
+//  Copyright © 2020 DZN. All rights reserved.
+//
+
+import UIKit
+
+/// The object that acts as the data source of the empty datasets.
+/// The data source must adopt the EmptyDataSetSource protocol. The data source is not retained. All data source methods are optional.
+public protocol EmptyDataSetSource: class {
+
+    /// Default is nil.
+    func title(forEmptyDataSet scrollView: UIScrollView) -> NSAttributedString?
+
+    /// Default is nil.
+    func description(forEmptyDataSet scrollView: UIScrollView) -> NSAttributedString?
+
+    /// Default is nil.
+    func image(forEmptyDataSet scrollView: UIScrollView) -> UIImage?
+
+    /// Default is nil.
+    func imagetintColor(forEmptyDataSet scrollView: UIScrollView) -> UIColor?
```

Comment:
Would definitely make this a camel case `imageTintColor`
```suggestion
    func imageTintColor(forEmptyDataSet scrollView: UIScrollView) -> UIColor?
```